src/authutil.ts

```diff
@@ -13,10 +20,19 @@ export function configAuthentication(registryUrl: string) {
     registryUrl += '/';
   }
 
-  writeRegistryToFile(registryUrl, npmrc);
+  writeRegistryToFile({registryUrl, fileLocation: npmrc, alwaysAuth});
 }
 
-function writeRegistryToFile(registryUrl: string, fileLocation: string) {
+interface WriteRegistryToFileParams {
```

Comment:
I think I'd rather not define interfaces for these - it feels more natural/readable to me with simple parameters.

Reply:
Yeah I can simplify it. The move to interfaces is just an opinion that parameters should be position independent so that adding new parameters doesn't necessarily affect existing call sites if you add them optionally. Since none of the values felt necessary to the identity of the function, they all felt like they could be ephemeral in the future. Again, just opinion.